examples/operator/templates/deployment.yaml

```diff
@@ -104,10 +104,17 @@ spec:
         runAsNonRoot: true
       serviceAccountName: {{ include "operator.fullname" . }}-controller-manager
       terminationGracePeriodSeconds: 10
+      topologySpreadConstraints:
+      - maxSkew: 1
+        topologyKey: kubernetes.io/hostname
+        whenUnsatisfiable: DoNotSchedule
       volumes:
       - configMap:
           name: {{ include "operator.fullname" . }}-manager-config
         name: manager-config
       - name: secret-volume
         secret:
-          secretName: {{ include "operator.fullname" . }}-secret-ca
\ No newline at end of file
+          secretName: {{ include "operator.fullname" . }}-secret-ca
+{{- if .Values.topologySpreadConstraints }}
+      topologySpreadConstraints: {{- include "tplvalues.render" (dict "value" .Values.topologySpreadConstraints "context" $) | nindent 8 }}
```

Comment:
Sorry @RealAnna i had to revert constraints feature in [this commit](https://github.com/arttor/helmify/commit/db73ef44705129b0663316226ca860865cd3fa74) because of duplicates in specs. Commit also contains a template for `nodeSelector` which can be used as an example for other specs.